plasmapy/formulary/collisions.py

```diff
@@ -2033,3 +2036,63 @@ def coupling_parameter(
         )
 
     return coulomb_energy / kinetic_energy
+
+
+class RelaxationRates:
+    @validate_quantities(
+        n_b={"can_be_negative": False},
+        T_b={"can_be_negative": False, "equivalencies": u.temperature_energy()},
+    )
+    def __init__(
+        self,
+        species: (particles.Particle, particles.Particle),
+        v_a: u.cm / u.s,
```

Comment:
I think v_a should be optional, and if not provided should be calculated using `thermal_speed` from T_a

Reply:
What about v_b?